### DISCUSSION TITLE
Welcome to alx-pre_course Discussions!

### DISCUSSION THREAD

**Discussion by Dine5** [Dine5/alx-pre_course#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yitbaa** [comment]
I don't have any clue on where to start, the time schedule, and everything. Clear points need to be made and described
